_includes/pbc/all/install-features/202505.0/install-the-spryker-core-feature.md

```diff
@@ -0,0 +1,1527 @@
+
+```php
+<?php
+
+use Spryker\Shared\Session\SessionConfig;
+use Spryker\Shared\Session\SessionConstants;
+use Spryker\Shared\SessionRedis\SessionRedisConfig;
+use Spryker\Shared\SessionRedis\SessionRedisConstants;
+
+// ---------- Session
+$config[SessionConstants::ZED_SESSION_SAVE_HANDLER] = SessionRedisConfig::SESSION_HANDLER_REDIS;
+$config[SessionRedisConstants::ZED_SESSION_TIME_TO_LIVE] = SessionConfig::SESSION_LIFETIME_1_HOUR;
+$config[SessionRedisConstants::LOCKING_TIMEOUT_MILLISECONDS] = 0;
+$config[SessionRedisConstants::LOCKING_RETRY_DELAY_MICROSECONDS] = 0;
+$config[SessionRedisConstants::LOCKING_LOCK_TTL_MILLISECONDS] = 0;
+```
+
+{% info_block warningBox "" %}
+
+`SessionRedisConfig::SESSION_HANDLER_REDIS_LOCKING` and `SessionRedisConfig::SESSION_HANDLER_REDIS` can be used as values for `SessionConstants::ZED_SESSION_SAVE_HANDLER`.
+
+{% endinfo_block %}
+
+In case of a multi-instance Redis setup, add the following configuration:
+
+**config/Shared/config_default.php**
+
+```php
+<?php
+
+use Spryker\Shared\SessionRedis\SessionRedisConstants;
+
+$config[SessionRedisConstants::ZED_SESSION_REDIS_DATA_SOURCE_NAMES] = ['tcp://127.0.0.1:10009', 'tcp://10.0.0.1:6379'];
+$config[SessionRedisConstants::ZED_SESSION_REDIS_CLIENT_OPTIONS] = [
+    'replication' => 'sentinel',
+    'service' => 'mymaster',
+    'parameters' => [
+        'password' => 'secret',
+        'database' => 2,
+    ],
+];
+```
+
+{% info_block warningBox "" %}
+
+This configuration is used exclusively. In other words, you can't use any other Redis configuration.
+
+{% endinfo_block %}
+
+In case of a single-instance Redis setup, add the following configuration:
+
+**config/Share/config_default.php**
+
+```php
+<?php
+
+use Spryker\Shared\SessionRedis\SessionRedisConstants;
+
+$config[SessionRedisConstants::ZED_SESSION_REDIS_SCHEME] = 'tcp';
+$config[SessionRedisConstants::ZED_SESSION_REDIS_HOST] = '127.0.0.1';
+$config[SessionRedisConstants::ZED_SESSION_REDIS_PORT] = 6379;
+$config[SessionRedisConstants::ZED_SESSION_REDIS_PASSWORD] = false;
+$config[SessionRedisConstants::ZED_SESSION_REDIS_DATABASE] = 2;
+```
+
+If the file system is used as a session storage, add the following configuration:
+
+**config/Shared/config_default.php**
+
+```php
+<?php
+
+use Spryker\Shared\Session\SessionConfig;
+use Spryker\Shared\Session\SessionConstants;
+use Spryker\Shared\SessionFile\SessionFileConfig;
+use Spryker\Shared\SessionFile\SessionFileConstants;
+
+// ---------- Session
+$config[SessionConstants::ZED_SESSION_SAVE_HANDLER] = SessionFileConfig::SESSION_HANDLER_FILE;
+$config[SessionFileConstants::ZED_SESSION_TIME_TO_LIVE] = SessionConfig::SESSION_LIFETIME_1_HOUR;
+$config[SessionFileConstants::ZED_SESSION_FILE_PATH] = session_save_path();
+```
+
+#### Configure SecurityBlocker
+
+1. `SecurityBlocker` stores information about blocked accounts in Redis. So, it needs connection information. You can get it in the environment configuration of your project:
```

Comment:
1. What blocked accounts are those?
2. Why does it need connection information because of blocked accounts?
3. Instructions on what to do are not clear; it's not clear what to do with the following code snippet